src/wallet/wallet.cpp

```diff
@@ -4098,12 +4097,12 @@ std::shared_ptr<CWallet> CWallet::CreateWalletFromFile(const std::string& name,
         int nMaxVersion = gArgs.GetArg("-upgradewallet", 0);
         if (nMaxVersion == 0) // the -upgradewallet without argument case
         {
-            LogPrintf("Performing wallet upgrade to %i\n", FEATURE_LATEST);
+            walletInstance->WalletLogPrintf("Performing wallet upgrade to %i\n", FEATURE_LATEST);
             nMaxVersion = CLIENT_VERSION;
             walletInstance->SetMinVersion(FEATURE_LATEST); // permanently upgrade the wallet immediately
         }
         else
-            LogPrintf("Allowing wallet upgrade up to %i\n", nMaxVersion);
+            walletInstance->WalletLogPrintf("Allowing wallet upgrade up to %i\n", nMaxVersion);
         if (nMaxVersion < walletInstance->GetVersion())
         {
             InitError(_("Cannot downgrade wallet"));
```

Comment:
There are a couple of logs lower down that you've missed:

- `            LogPrintf("Upgrading wallet to HD\n");` (L4131)
- `            LogPrintf("Upgrading wallet to use HD chain split\n");` (L4143)

Reply:
Fixed

Reply:
Is this one fixed?